Makefile

```diff
@@ -20,6 +20,13 @@ build: build-ui
 	go mod vendor
 	go build -ldflags "-s -w" main.go
 
+build-go-for-docker:
+	go mod vendor
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags "-s -w" -o main main.go
```

Comment:
We don't use cgo here at all, pls remove

Reply:
I agree, tho when running the project on Alpine it fails.
The `CGO` is also used here: https://github.com/nfx/slrp/blob/main/.goreleaser.yaml#L13.
This topic specifically was previously discussed here: https://github.com/nfx/slrp/pull/90#discussion_r1210843210
> "...but slrp must be build with static linkage CGO_ENABLED=0 I believe..."

wdyt?

Reply:
Here's the situation where a code comment could have avoided a review comment 😉

Reply:
Will add the comment where needed ;)